лаб 1.py

```diff
@@ -0,0 +1,100 @@
+import doctest
+
+
+class Printer:
+    def init(self, brand: str, max_pages_per_minute: int):

```

Comment:
```suggestion
class Printer:
    """
    ...
    """
    def __init__(self, brand: str, max_pages_per_minute: int):

```
Метод инициализации класса вызван не верно, а также необходимо написать документацию также для самого класса. Сейчас у верно написана документация для методов, а у классов она отсутствует полностью.